test/src/test/testtool/XsltProviderListener/scenario04.properties

```diff
@@ -1,7 +1,7 @@
 scenario.description = Test the XSLT provider listener with xslt parameters with xslt file in configuration folder
 
 java.XsltProviderListener.className = org.frankframework.larva.XsltProviderListener
-java.XsltProviderListener.filename = configurations/UTIL/Larva/xsltProviderListener/ApplyValueOverrides.xslt
```

Comment:
Deze moet wel echt as is blijven. Dat is het hele punt van deze test. (lees ook de description)